pkg/kp/flags/kingpin.go

```diff
@@ -19,6 +20,9 @@ func ParseWithConsulOptions() (string, kp.Options) {
 	headers := kingpin.Flag("header", "An HTTP header to add to requests, in KEY=VALUE form. Can be specified multiple times.").StringMap()
 	https := kingpin.Flag("https", "Use HTTPS").Bool()
 	wait := kingpin.Flag("wait", "Maximum duration for Consul watches, before resetting and starting again.").Default("30s").Duration()
+	caFile := kingpin.Flag("tls-ca-file", "File containing the x509 PEM-encoded CA ").ExistingFile()
```

Comment:
Having "tls-" seems wordy, and I don't want to ever type that.  What do you think about removing that prefix? Incidentally, would that make these flags the same as for curl?